test/core.vwtest.json

```diff
@@ -5486,5 +5472,71 @@
     "input_files": [
       "train-sets/explore_eval_egreedy.dat"
     ]
+  },
+  {
+    "id": 425,
+    "desc": "Spin off automl model to lower bitsize to cb_explore",
```

Comment:
add cubic test, or is this only for quadratics for now?

Reply:
Cubics should work, I can add a test for that

Reply:
add a unit test here
https://github.com/VowpalWabbit/vowpal_wabbit/pull/4279/files#diff-9932fc58f28480ef64a94f6fc4cb26b0b50e7d94c621a3edb00fd0131fcc14dfR400